src/test/crimson/test_crimson_scrub.cc

```diff
@@ -853,7 +853,7 @@ TEST_P(TestSingleError, SingleError) {
   bool found_selected_oi = false;
   for (const auto &shard : shards) {
     auto siter = obj_error.shards.find(
-      librados::osd_shard_t(shard.osd, shard.shard)
+      librados::osd_shard_t{shard.osd, shard.shard}
```

Comment:
not related to this commit.

Reply:
But won't compile in 'ci' otherwise. It's in a separate commit.

Reply:
Hmm, what change made this required?

Reply:
It caused a compilation warning, but -Werror is turned on for some builds (why???).
I'll try to reconstruct the issue, to list the specific build.